### DISCUSSION TITLE
Why the SnapshotTesting.podspec is removed?

### DISCUSSION THREAD

**Discussion by emarashliev** [pointfreeco/swift-snapshot-testing#754] [Q&A]
Why has the SnapshotTesting.podspec file been removed from the repo? What's the point? Did you consider that you can break someone's project before taking this decision?

**Comment by stephencelis** [comment]
@emarashliev We use issues for bug tracking, so I'm going to convert this to a discussion.

We're not aware of how discontinuing future support for CocoaPods can "break someone's project." The podspec still exists for 1.9.0 and earlier:

https://github.com/CocoaPods/Specs/blob/master/Specs/5/6/8/SnapshotTesting/1.9.0/SnapshotTesting.podspec.json

And you are free to maintain your own podspec for more recent versions of SnapshotTesting should you want to integrate it into your CocoaPods flow. (Otherwise you can use SPM, which is widely available and integrates directly into tools like Swift and Xcode.)

Here are some related discussions:

- https://github.com/pointfreeco/swift-snapshot-testing/discussions/729
- https://github.com/pointfreeco/swift-snapshot-testing/issues/644
